datasets/gov/aaa/isleid_perd_tersalai.csv

```diff
@@ -0,0 +1,73 @@
+,,,,,year,integer,,,,2,open,,Ataskaitiniai metai,
+,,,,,,comment,type,,"update(type: ""date"", ref: ""Y"")",4,,spinta:213,,
+,,,,,,,,,,,,,,
+,,,,OffsiteWasteTransfer,,,,,,4,,,Atliekų perdavimas už objekto ribų,
+,,,,,id,string,,,,4,open,,Perduodamų atliekų Id,Perduodamų atliekų identifikacinis numeris (Eil. Nr.)
+,,,,,localid,string,,,,4,open,,Gamybos įrengimų komplekso Id,Gamybos įrengimų komplekso identifikacinis numeris
+,,,,,wasteclassification,string,,,,4,open,,Atliekų klasifikacija,Atliekų klasifikacija
+,,,,,wastetreatment,string,,,,4,open,,Atliekų naudojimas,Atliekų pašalinimas/perdirbimas
+,,,,,quantity,number,t,,,4,open,,Visos atliekos,"Visų atliekų kiekis, tonomis per ataskaitinius metus"
+,,,,,methodcode,string,,,,4,open,,Metodo kodas,Metodo kodas (apskaičiuota/išmatuota reikšmė)
+,,,,,methodclassification,string,,,,4,open,,Metodo klasifikacija,Metodo klasifikacija
+,,,,,nameofreceiver,string,,,,4,open,,Gavėjo pavadinimas,Atliekų gavėjo pavadinimas
+,,,,,countrycode,string,,,,4,open,,Gavėjo šalies kodas,Pagal https://dd.eionet.europa.eu/vocabulary/EPRTRandLCP/CountryCodeValue/view
+,,,,,year,integer,,,,2,open,,Ataskaitiniai metai,
+,,,,,,comment,type,,"update(type: ""date"", ref: ""Y"")",4,,spinta:213,,
+,,,,,,,,,,,,,,
+,,,,PollutantRelease,,,,,,4,,,Išleidžiamų teršalų duomenys,
+,,,,,id,string,,,,4,open,,Išleidžiamų teršalų Id,Išleidžiamų teršalų identifikacinis numeris (Eil. Nr.)
+,,,,,localid,string,,,,4,open,,Gamybos įrengimų komplekso Id,Gamybos įrengimų komplekso identifikacinis numeris
+,,,,,pollutant,string,,,,4,open,,Teršalas,"Teršalo kodas, pateiktas aktyvia nuoroda pagal https://dd.eionet.europa.eu/vocabulary/EPRTRandLCP/EPRTRPollutantCodeValue/view"
+,,,,,mediumcode,string,,,,4,open,,Terpės kodas,Terpė (oras/vanduo) į kurią išleidžiamas teršalas
+,,,,,quantity,number,kg,,,4,open,,Visas teršalo kiekis,"Teršalo kiekis, kilogramais per ataskaitinius metus"
+,,,,,methodcode,string,,,,4,open,,Metodo kodas,Metodo kodas (apskaičiuota/išmatuota/numatyta reikšmė)
+,,,,,methodclassification,string,,,,4,open,,Metodo klasifikacija,Metodo klasifikacija
+,,,,,year,integer,,,,2,open,,Ataskaitiniai metai,
+,,,,,,comment,type,,"update(type: ""date"", ref: ""Y"")",4,,spinta:213,,
+,,,,,,,,,,,,,,
+,,,,ProductionFacilityReport,,,,,,4,,,,
+,,,,,localid,string,,,,4,open,,Gamybos įrengimų kompleksų duomenys,
+,,,,,namespace,string,,,,4,open,,Gamybos įrengimų komplekso Id,Gamybos įrengimų komplekso identifikacinis numeris
+,,,,,reportid,integer,,,,4,open,,Ataskaitos Id,Ataskaitos identifikacinis numeris
+,,,,,year,integer,,,,2,open,,Ataskaitiniai metai,
+,,,,,,comment,type,,"update(type: ""date"", ref: ""Y"")",4,,spinta:213,,
+,,,,,,,,,,,,,,
+,,,,ProductionInstallationPartReport,,,,,,4,,,Gamybos įrengimų duomenys,
+,,,,,localid,string,,,,4,open,,Gamybos įrengimo Id,Gamybos įrengimo identifikacinis numeris
+,,,,,namespace,string,,,,4,open,,Duomenis teikianti šalis,Nuoroda į duomenis teikiančią šalį
```

Comment:
Ar tai standartinis šalies trumpinio pateikimas? Ką reiškia dalis po taško?
![image](https://user-images.githubusercontent.com/85924252/226911045-412eff38-8643-4988-9a24-a358f7384cbe.png)

Reply:
Patikslinimas iš atstovo: "kadangi teikiame šias ataskaitas Europos komisijai, jų duomenų bazėse reikia įrašyti šalies, atsakingos už duomenų pateikimą, kodą. Tad LT.CAED ir yra tas kodas, kuris parodo kad duomenis pateikia Lietuvos atsakingoji institucija (LT.CAED – Lithuanian Competent Authority for Environmental Data).
Nuorodų kaip ir nėra papildomų, nes kiekviena šalis tiesiog turi panašų kodą."